### DISCUSSION TITLE
How to use deliberation encoder based 2 pass model in SLU task

### DISCUSSION THREAD

**Discussion by Huang0301** [espnet/espnet#4581] [Q&A] [answered]
https://arxiv.org/abs/2207.06670
Hi, I want to implement the 2 pass model in this paper with espnet2, but I didn't find any relevant code. Does espnet2 support implementing this 2 pass model now? how should I implement it? 
thank you!

**Comment by sw005320** [comment, marked as answer]
Yes, you can find it in @siddhu001's PR https://github.com/espnet/espnet/pull/4569

**Reply by Huang0301** [reply]
Thank you!
